content/guides/models/registry/configure_registry.md

```diff
@@ -9,6 +9,53 @@ weight: 3
 
 A registry admin can [configure registry roles]({{< relref "configure_registry.md#configure-registry-roles" >}}), [add users]({{< relref "configure_registry.md#add-a-user-or-a-team-to-a-registry" >}}), or [remove users]({{< relref "configure_registry.md#remove-a-user-or-team-from-a-registry" >}}) from a registry by configuring the registry's settings.
 
+## Diagram
+
+This diagram illustrates the hierarchical permission structure in Weights & Biases (W&B), showing the relationships between Organization, Team, and Registry roles and how permissions are inherited. Users receive the highest permission level between their individual assignment and team membership, with Registry roles limited to three fixed types (Admin, Member, Viewer) that determine what actions users can perform. 
+
+{{< alert >}}
+The permissions for teams and registries are separate. For example, being an admin on a team does not make a user an admin on a registry.
+{{< /alert >}}
+
+```mermaid
```

Comment:
Changing the docs to dark mode makes the font in the diagram unreadable :(. 

Looking at the mermaid docs, it looks like we can use a themes to get our desired output. Something like this might work?

```
%%{init: {'theme':'neutral'}}%%
```